sample/redo/lib/main.dart

```diff
@@ -17,6 +17,7 @@ String addPrefix(int id) => '$prefix$id';
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    Replicache.setLogLevel(LogLevel.debug);
```

Comment:
[cbb] Seems better to do this outside the build method.

Reply:
Crap I didn't mean to check this in.